### ISSUE TITLE
odd scenarios can trigger aframe-player tag to highjack focus

### ISSUE THREAD

**Issue by btopro** [LRNWebComponents/hax-body#134]
It's very difficult to replicate and isn't really related to HAX (but since it's an element we demo should figure out how to resolve) but some reason after a `<a-frame>` tag has made its way onto the DOM if your using hax and do the following:
- activate developer mode
- insert a layout that has a paragraph placeholder
- double click it (which escalates to paragraph
- click the paragraph and start typing
- now aframe has locked the page and scrolled back tot he top unless you hit esc. it's somehow gotten focus

**Comment by heyMP** [comment]
Just a guess but I think this might fix it. https://github.com/aframevr/aframe/blob/master/docs/components/embedded.md

**Comment by btopro** [comment]
now doesn't even seem to work in latest version of chrome unless I'm missing something

**Comment by btopro** [comment]
that lead me to the fix; thx @heyMP
